AGENTS.md

```
@@ -2,59 +2,57 @@
 Black lives matter. Support the Equal Justice Initiative. ✊🏽✊🏾✊🏿
 
-Gmail+Trello integration. Extension for Chrome browser and other developments.
+Gmail+Trello integration. An extension for the Chrome browser and other developments.
 
-Published on Chrome Web Store: https://g2t.pub/chrome
-
-Support, commonly asked questions, interaction with other fans: https://g2t.support
+Published on Chrome Web Store: [Gmail-2-Trello](https://g2t.pub/chrome)
 
+Support, common questions, interaction with other fans: [g2t.support](https://g2t.support)
 ## Agent Instructions
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)